.gitignore

```diff
@@ -1,2 +1,4 @@
 *.pyc
-secrets/
\ No newline at end of file
+secrets/**
```

Comment:
Currently I'm storing the GEE service account key and Google Cloud details in there, but they definitely don't need to be in the repo!

> Is there a way to store the key that GEE prompts you for in here?

Good question! The keys you enter in `ee.Authenticate` are very short-lived, but once you're authenticated it does store a longer-term key at `~/.config/earthengine`, which is where `ee.Initialize` looks. I experimented with copying that key into the Docker container, but found it was a little trickier than expected, especially if the key didn't already exist. I'm sure it could be done with a Bash script though. The other possible complication is that the container would always run with whatever EE account was authenticated locally, but I'm not sure that's really an issue. 

Having to authenticate every time is definitely a nuisance, so I think I'll open an issue to take another look at copying credentials into the container.